problems/5.loops.js

```diff
@@ -1,19 +1,26 @@
 /** Returns every even number from 0 to a given number
 * @param {number} num - The number to iterate up to
-* @param {number[]} - An array of even numbers from 0 to {num}
+* @returns {number[]} - An array of even numbers from 0 to {num}
```

Comment:
Look at the type of data you're taking in. num is a number NOT an array. There is no num.length or num[i] because it's a number.